kafka-connectors/kafka010/src/main/java/org/apache/apex/malhar/kafka/AbstractKafka010InputOperator.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.apex.malhar.kafka;
+
+import java.util.Properties;
+
+public abstract class AbstractKafka010InputOperator extends AbstractBaseKafkaInputOperator
```

Comment:
I feel, it should be easy for user to move from 0.9 to 0.10 version by just changing the application pom.xml , If the class name is different then source code change is also needed , @tweise what do you think about following class structures.

- AbstractKafkaInputOperator  (In malhar-kafka010 artifact for 0.10 kafka)
- AbstractKafkaInputOperator (In malhar-kafka artifact for 0.9 kafka)
- AbstractBaseKafkaInputOperator (in malhar-kafka-common artifact)